api-compact.yaml

```diff
@@ -199,3 +274,45 @@ endpoints:
           x-amazon-apigateway-integration:
             <<: *apigateway_integration_with_id_and_querystring_parameters
             uri: "https://${stageVariables.enterprise_host}/enterprise/api/v1/catalogs/{id}/courses/"
+
+    # /v1/enterprise_catalogs/
```

Comment:
@UmanShahzad We still have underscores in L278, L292, L294, L305. Should probably turn them into dashes after fbdf6222b8925de0ed2b4d2d302d014fb03fbdd4.